ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ClaimMapper.java

```diff
@@ -91,6 +93,10 @@ public void to(Claim claim, CCDCase.CCDCaseBuilder builder) {
             .feeAmountInPennies(claimData.getFeeAmountInPennies());
     }
 
+    private boolean isLeadApplicant(Claim claim, AtomicInteger applicantIndex) {
+        return !claim.getClaimData().isClaimantRepresented() && applicantIndex.getAndIncrement() == 0;
```

Comment:
why do we `getAndIncrement` here? to me this means the method is doing more than just checking if `isLeadApplicant` so maybe rename.

Reply:
agree, missed in refactoring. 👍